pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -249,6 +249,9 @@ func (s *Summarizer) processOneResultFileForHost(results *kb.Controls, hostname
 				continue
 			}
 
+			if check.Type == CheckTypeSkip {
+				check.State = NA
```

Comment:
If the skip from yaml is considered as NA, we should update the remediation message to reflect as to why it's being skipped?

Reply:
yes, good point, I have appended the reasons from KDM to the remediations